ICSharpCode.Decompiler.Tests/TestCases/Correctness/InitializerTests.cs

```diff
@@ -375,6 +375,18 @@ public static void StructInitializerWithinObjectInitializer()
 			});
 		}
 
+#if !NET40 && ROSLYN
+		public static ReadOnlySpan<byte> ReadOnlySpanInitializer_ByteArray()
+		{
+			return new byte[] { 1, 2, 3 };
+		}
+
+		public static ReadOnlySpan<int> ReadOnlySpanInitializer_Int32Array()
+		{
+			return new int[] { 1, 2, 3 };
+		}
+#endif
+
```

Comment:
Should I have put these in a new file and modified the options? It currently fails for Roslyn v1.

Reply:
Is there a reason why these tests are "correctness" tests and not "pretty" tests?

It seems like this is a relatively new language feature... you can simply use the `CSx` preprocessor symbol that matches the earliest version of C# where this feature is supported.